src/main/java/de/doubleslash/keeptime/view/ManageProjectController.java

```diff
@@ -0,0 +1,88 @@
+package de.doubleslash.keeptime.view;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import de.doubleslash.keeptime.model.Model;
+import javafx.fxml.FXML;
+import javafx.scene.control.CheckBox;
+import javafx.scene.control.ColorPicker;
+import javafx.scene.control.Spinner;
+import javafx.scene.control.SpinnerValueFactory.IntegerSpinnerValueFactory;
+import javafx.scene.control.TextArea;
+import javafx.scene.control.TextField;
+import javafx.scene.layout.GridPane;
+import javafx.scene.paint.Color;
+
+public class ManageProjectController {
+
+   private static final Logger LOG = LoggerFactory.getLogger(ManageProjectController.class);
+
+   private Model model;
+
+   @FXML
+   private GridPane grid;
+
+   @FXML
+   private TextField nameTextField;
+
+   @FXML
+   private TextArea descriptionTextArea;
+
+   @FXML
+   private ColorPicker textFillColorPicker;
+
+   @FXML
+   private CheckBox isWorkCheckBox;
+
+   @FXML
+   private Spinner<Integer> sortIndexSpinner;
+
+   public GridPane getGrid() {
+      return grid;
+   }
+
+   public TextField getNameTextField() {
+      return nameTextField;
+   }
+
+   public TextArea getDescriptionTextArea() {
+      return descriptionTextArea;
+   }
+
+   public ColorPicker getTextFillColorPicker() {
+      return textFillColorPicker;
+   }
+
+   public CheckBox getIsWorkCheckBox() {
+      return isWorkCheckBox;
+   }
+
+   public Spinner<Integer> getSortIndexSpinner() {
+      return sortIndexSpinner;
+   }
+
+   public void setModel(final Model model) {
+      this.model = model;
+   }
+
+   public void setValues(final String name, final String description, final Color textFill, final boolean work,
```

Comment:
as this controller is currently represents "Project", it could also take the Project object as parameter. As well it can return a Project -object, if the values are requested

Reply:
Done.

Reply:
get is missing :)

Reply:
Done.

Reply:
get should also be used in the classes -- i changed it